### PR TITLE
Create es.ftl

### PR DESCRIPTION
**Pull request by Lord-Galahad**
Add translation to spanish

### CODE REVIEW

File: resources/es.ftl [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+welcome_has_auth = ¡Bienvenido, {$mention}! Estás autenticado como [{$name}](<{$user_link}>), no necesitas autenticarte de nuevo.
+
+welcome_has_auth_failed = ¡Bienvenido, {$mention}! Estás autenticado (oopsie, no tengo información al respecto), no necesitas autenticarte de nuevo.
+
+welcome = ¡Bienvenido, {$mention}! Si deseas autenticar (validar) tu identidad como integrante del movimiento Wikimedia por favor usa: </auth:1221128504410898571>
+
+whois_global_groups = Grupos globales: {$groupslist}
+
+whois_blocked = **BLOQUEADO**
+whois_locked = **LOCKED**
```
Comment (fee1-dead) [inline resources/es.ftl]: This is the message when someone's account is locked. Is this the same in spanish?

File: resources/es.ftl [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+whois_edits = Ediciones: {$edits}
+whois_groups = Grupos: {$groupslist}
+whois_overflow = Limitado a 10 proyectos. Da clic en su nombre al principio para observar toda la información.
+whois_no_block_reason = <!-- No reason given -->
```
Comment (fee1-dead) [inline resources/es.ftl]: This is the message when someone is blocked but the reason is empty. Could you please translate this?

File: resources/es.ftl [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+# If you need a different date format other than YYYY-MM-DD, let me know.
+whois = Discord: {$mention}
+    Fecha de registro: {$registration}
+    Proyecto principal: {$home}
+    {$global_groups}Ediciones totales: {$edits}
+
+# These are currently unused for now.
+server_auth_success = Success! Authorization information sent to the bot :)
+server_auth_expired = Auth request was expired or invalid.
```
Comment (fee1-dead) [inline resources/es.ftl]: although currently unused, we might use them in the future. could you please translate these as well? Thanks!

**Review by fee1-dead — COMMENTED** [review]
Thanks! Left a few comments for messages that you might want to translate.

File: resources/es.ftl [hunk 4]
```diff
@@ -0,0 +1,59 @@
+auth = Por favor usa el siguiente enlace para autenticarte en tu cuenta Wikimedia: [Authenticate]({$url})
+
+auth_exists_in_server = Actualmente estás autenticado en este servidor. No es necesario autenticarse de nuevo.
+
+auth_to_server = Estás identificado como [{$name}](<{$url}>). ¿Quieres autenticarte de nuevo en este servidor?
+yes = sí
+no = No
```
Comment (fee1-dead) [inline resources/es.ftl]: Can si be capitalized? this is the text on buttons so consistency would be good.

File: resources/es.ftl [hunk 5]
```diff
@@ -0,0 +1,59 @@
+auth = Por favor usa el siguiente enlace para autenticarte en tu cuenta Wikimedia: [Authenticate]({$url})
```
Comment (fee1-dead) [inline resources/es.ftl]: Authenticate should ideally be translated too. (or is this the same word in spanish?)

File: resources/es.ftl (hunk 1 above)
Comment (Lord-Galahad) [reply resources/es.ftl]: Done, modified

**Comment by Lord-Galahad** [comment]
Done with the comments. Please let me know if you need anything else. :)

**Comment by fee1-dead** [comment]
Thanks for taking your time and contributing :)

File: resources/es.ftl [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+auth_to_server = Estás identificado como [{$name}](<{$url}>). ¿Quieres autenticarte de nuevo en este servidor?
+yes = Sí
+no = No
+
+auth_footer = Este enlace tendrá validez por 5 minutos.
+
+authreq_canceled = Autenticación cancelada.
+authreq_expired = Autenticación expirada.
+
+authlog = {$mention} autenticado como [User:{$username}](<{$user_link}>) (id {$wmf_id}).
```
Comment (fee1-dead) [inline resources/es.ftl:14]: Oh, also. If you don't want to use the `User:` prefix, you can remove it like `pl` did, or you could translate it into the prefix that your language uses.

**Pull request merged**